samples/WOCCatalog/WOCCatalog/AccelerateViewController.m

```diff
@@ -326,8 +326,10 @@ - (UIImage*)transformImage:(UIImage*)image {
     // create CGImageRef from vImage_Buffer output
     CGColorSpaceRef colorSpace = CGColorSpaceCreateDeviceRGB();
 
-    CGContextRef ctx = CGBitmapContextCreate(
-        outBuffer.data, outBuffer.width, outBuffer.height, 8, outBuffer.rowBytes, colorSpace, (CGBitmapInfo)kCGImageAlphaNoneSkipLast);
+    const CGBitmapInfo bitmapInfo = kCGImageAlphaNoneSkipLast | kCGBitmapByteOrder32Little;
```

Comment:
BitmapInfo (including byteOrder flags) is intended to indicate what the underlying format is. The way it is processed is:
- A byte order of 32big is assumed (ie RGB)
- Use alphaInfo to determine: whether there is an alpha component or padding variable present and if so what its position is and place it in the pixel. In this case and at this point we would have: RGBX
- Use the byteOrder to determine if it is different from the default value and whether or not the order needs to be flipped. In this case, kCGBitmapByteOrder32Little would indicate that the byte order would need to be flipped so RGBX becomes XBGR.